pkg/refactor/eventstore/eventstore.go

```diff
@@ -0,0 +1,81 @@
+package eventstore
+
+import (
+	"context"
+	"time"
+
+	cloudevents "github.com/cloudevents/sdk-go/v2"
+	"github.com/google/uuid"
+)
+
+type CloudEventsStore interface {
+	Create(ctx context.Context,
+		nsID uuid.UUID,
+		fireTime time.Time,
+		event *cloudevents.Event,
+		eventID uuid.UUID,
+		processed bool) error
```

Comment:
We can remove this argument. I don't believe the events logic will ever create an event as already processed. That only happens with an update later.